extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -208,7 +209,8 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   scrollHandler_() {
-    const currentScrollTop = this.element./*OK*/scrollTop;
+    // If scroll top is 0, it's set to 1 to avoid scroll-freeze issue.
+    const currentScrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
Hi @muxin, actually `this.element./*OK*/scrollTop = currentScrollTop` were needed. I just checked it on safari browser and it finally works.